pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -270,6 +270,64 @@ func (c *Cluster) startProvisioning(config *provisionConfig) {
 			}
 		}
 	}
+
+	// Deploying test based on the storageclassDeviceSet
+	// Todo: Replace the below logic to deploy osd pods over pvcs
+	for setIndex, storageClassDeviceSet := range c.DesiredStorage.StorageClassDeviceSets {
+		for i, pvcTemplate := range storageClassDeviceSet.VolumeClaimTemplates {
+			pvcStorageClassDeviceSetPVCId := fmt.Sprintf("%s-%v-%v", storageClassDeviceSet.Name, setIndex, i)
+			pvcStorageClassDeviceSetPVCIdLabelSelector := fmt.Sprintf("rook.ceph.io/StorageClassDeviceSetPVCId=%s", pvcStorageClassDeviceSetPVCId)
+			pvcLables := map[string]string{
+				"rook.ceph.io/storageClassDeviceSet":      storageClassDeviceSet.Name,
```

Comment:
how about a namespace of `ceph.rook.io`? we already have that precedent in other places in rook.

Reply:
yup changed